curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f0286404aefb0562a4fdf9.md

```diff
@@ -54,31 +54,36 @@ assert.equal(heads?.length, 1);
 Your `link` element should be a self-closing element.
 
 ```js
-assert(code.match(/<link[\w\W\s]+\/>/i));
+assert(!code.includes("</link"));
```

Comment:
```suggestion
assert(code.match(/<link[\w\W\s]+\/>/i));
```
The changes meant that it only checks for the absence of a particular mistake rather than the presence of a self-closing element.

Reply:
So what should this test case be? I presume we just want to prevent users from doing something like this
`<link ></link>`

Reply:
Sorry for the delay getting back to you.  I think `/<link[^>]+\/>/` should do the trick.  It rejects code like `<link></link>` and `<link><link/>` which are likely to be common mistakes.